Adding labels while processing a template with no object labels panics

```
[vagrant@openshiftdev origin]$ osc process -f test/templates/fixtures/guestbook.json -l app=guestbook
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/openshift/origin/pkg/cmd/cli/cmd.RunProcess(0xc208303580, 0x7ffa646761f0, 0xc20804e008, 0xc2082dc820, 0xc2082bfb80, 0x0, 0x4, 0x0, 0x0)
	/home/mkargaki/go/src/github.com/kargakis/origin/_output/local/go/src/github.com/openshift/origin/pkg/cmd/cli/cmd/process.go:185 +0x903
github.com/openshift/origin/pkg/cmd/cli/cmd.func·015(0xc2082dc820, 0xc2082bfb80, 0x0, 0x4)
	/home/mkargaki/go/src/github.com/kargakis/origin/_output/local/go/src/github.com/openshift/origin/pkg/cmd/cli/cmd/process.go:73 +0x5f
github.com/spf13/cobra.(*Command).execute(0xc2082dc820, 0xc2082bfac0, 0x4, 0x4, 0x0, 0x0)
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/spf13/cobra/command.go:476 +0x65c
github.com/spf13/cobra.(*Command).Execute(0xc2082f9860, 0x0, 0x0)
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/spf13/cobra/command.go:541 +0x18d
main.main()
	/home/mkargaki/go/src/github.com/kargakis/origin/_output/local/go/src/github.com/openshift/origin/cmd/openshift/openshift.go:20 +0x147

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 6 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x31e1ce0)
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x78
created by github.com/golang/glog.init·1
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2a7

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 35 [select]:
github.com/GoogleCloudPlatform/kubernetes/pkg/util.(*tickRateLimiter).step(0xc2083a9e60, 0x435d70)
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/util/throttle.go:102 +0xfc
github.com/GoogleCloudPlatform/kubernetes/pkg/util.(*tickRateLimiter).run(0xc2083a9e60)
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/util/throttle.go:95 +0x28
created by github.com/GoogleCloudPlatform/kubernetes/pkg/util.NewTokenBucketRateLimiter
	/home/mkargaki/go/src/github.com/kargakis/origin/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/util/throttle.go:50 +0x81
```

--- a/pkg/cmd/cli/cmd/process.go
+++ b/pkg/cmd/cli/cmd/process.go
@@ -181,6 +181,9 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		if err != nil {
 			return err
 		}
+		if templateObj.ObjectLabels == nil {
+			templateObj.ObjectLabels = make(map[string]string)
+		}
 		for key, value := range lbl {
 			templateObj.ObjectLabels[key] = value
 		}

